src/models/config.ts

```diff
@@ -0,0 +1,7 @@
+export interface VoyagerConfig {
+  showDataSourceSelector?: boolean;
```

Comment:
I think this option looks reasonable for now, but longer term we might want to think how this interacts with the `data` parameter.  (If the data is provided from external app, do we need to turn this flag on automatically, etc.)